code/controllers/subsystem/vote.dm

```diff
@@ -407,3 +421,21 @@ SUBSYSTEM_DEF(vote)
 
 	if(SSvote)
 		SSvote.browse_to(client)
+
+/**
+  * Determine if the station has power.
+  *
+  * Actually, we'll just cheat and see if the Particle Accelerator is active.
+  * If the Particle Accelerator is active, we'll assume that we have power.
+  * See: https://www.youtube.com/watch?v=-dJolYw8tnk
+  */
+/proc/i_have_the_power()
+	var/obj/machinery/particle_accelerator/control_box/the_pa = locate("particle_accelerator_control_box")
+	if(!the_pa)
+		log_and_message_admins("Unable to locate Particle Accelerator Control Console on this map.")
+		log_and_message_admins("Missing tag: particle_accelerator_control_box")
+		return FALSE
+	return the_pa.active
```

Comment:
Should this be TRUE?

Reply:
It depends on how we want to fail.
`return FALSE` --> If we can't find a PA control box, assume that we DON'T have power (and hold the supercharge vote)
`return TRUE` --> If we can't find a PA control box, assume that we DO have power (and skip the supercharge vote)
Here I've opted to fail in the direction of holding a vote, as it gives more flexibility to the players.

The `return the_pa.active` is simply returning if the PA has been turned on or not.
This maps exactly to what we want.
Inactive PA (FALSE) --> Hold the supercharge vote
Active PA (TRUE) --> Skip the supercharge vote

Reply:
Hey, look at that, your log_and_message_admins() lines already say what you just said - if only I had read them @__@

Thank you, this makes sense ^__^